### PR TITLE
MCLOUD-6683: BUNDLE-2670: Braintree VBE - Virtual Terminal Page errors and will not load

### PR DESCRIPTION
**Pull request by magterskine**
### Description:
https://jira.corp.magento.com/browse/MCLOUD-6683:
Adds patch to fix issue for Braintree VBE where Braintree Virtual Terminal page throws console errors and displays loading spinner forever

### Fixed Issues:
https://jira.corp.magento.com/browse/BUNDLE-2670

### Manual Testing Scenarios
1. Enable Braintree
2. Navigate to: Sales > Braintree Virtual Terminal
3. Page should load correctly without errors

### CODE REVIEW

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
The is no proper description for the PR - please add one with correct format and full description
Tests are failing

**Comment by viktym** [comment]
Please, fix patch filename (use __ after ticket number and before the version):
```
  Correct format: <TICKET_NUMBER>__<TITLE>__<PACKAGE_VERSION>.patch                                 
  Example: MAGECLOUD-2899__fix_redis_slave_configuration__2.3.0.patch  
```

**Comment by shiftedreality** [comment]
@viktym @magterskine please re-create this PR into https://github.com/magento/quality-patches

**Review by BarnyShergold — DISMISSED** [review]

File: patches/BUNDLE-2670__braintree_virtual_terminal_ui_fix__4.1.0.patch [hunk 1]
```diff
@@ -0,0 +1,132 @@
+diff --git a/vendor/paypal/module-braintree-core/view/adminhtml/web/css/source/_module.less b/vendor/paypal/module-braintree-core/view/adminhtml/web/css/source/_module.less
+new file mode 100755
+index 0000000..efb2e66
```
Comment (NadiyaS) [inline patches/BUNDLE-2670__braintree_virtual_terminal_ui_fix__4.1.0.patch]: Please remove index. It is redundant information.

File: patches/BUNDLE-2670__braintree_virtual_terminal_ui_fix__4.1.0.patch [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+@@ -61,6 +61,9 @@ define([
+                     authorization: self.clientToken
+                 }, function (clientErr, clientInstance) {
+                     if (clientErr) {
++                        alert({
++                            content: $t('Please configure your Braintree Payments account in order to use the virtual terminal.')
++                        });
+                         console.error('Error!', clientErr);
+                         return self.error(response.clientErr);
+                     }
```
Comment (NadiyaS) [inline patches/BUNDLE-2670__braintree_virtual_terminal_ui_fix__4.1.0.patch]: Please add an empty line at the end

File: patches/BUNDLE-2670__braintree_virtual_terminal_ui_fix__4.1.0.patch [hunk 3]
```diff
@@ -0,0 +1,130 @@
+diff --git a/vendor/paypal/module-braintree-core/view/adminhtml/web/css/source/_module.less b/vendor/paypal/module-braintree-core/view/adminhtml/web/css/source/_module.less
```
Comment (shiftedreality) [inline patches/BUNDLE-2670__braintree_virtual_terminal_ui_fix__4.1.0.patch]: Please use `diff -Naur`

File: patches/BUNDLE-2670__braintree_virtual_terminal_ui_fix__4.1.0.patch [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
++
++.hosted-error {
++    clear: both;
++    color: @braintree-error__color;
++    height: 0;
++    opacity: 0;
++}
+
+
+diff --git a/vendor/paypal/module-braintree-core/view/adminhtml/web/js/virtual.js b/vendor/paypal/module-braintree-core/view/adminhtml/web/js/virtual.js
```
Comment (shiftedreality) [inline patches/BUNDLE-2670__braintree_virtual_terminal_ui_fix__4.1.0.patch]: Please use `diff -Naur`

**Review by shiftedreality — CHANGES_REQUESTED** [review]

File: patches/BUNDLE-2670__braintree_virtual_terminal_ui_fix__4.1.0.patch (hunk 2 above)
Comment (magterskine) [reply patches/BUNDLE-2670__braintree_virtual_terminal_ui_fix__4.1.0.patch]: Done

**Review by NadiyaS — APPROVED** [review]

**Review by shiftedreality — APPROVED** [review]

**Comment by BaDos** [comment]
FT fails:

```Applying patch BUNDLE-2670 (/app/vendor/magento/magento-cloud-patches/patches/BUNDLE-2670__braintree_virtual_terminal_ui_fix__4.1.0.patch) failed.
Error Output:
================
error: vendor/paypal/module-braintree-core/view/adminhtml/web/css/source/_module.less: already exists in working directory
Patch BUNDLE-2670 can't be applied to clean Magento instance
[2020-08-07 17:19:09] ERROR: [12] The command "php ./vendor/bin/ece-patches apply --no-interaction" failed. Error: patch conflict happened
Applying patch BUNDLE-2670 (/app/vendor/magento/magento-cloud-patches/patches/BUNDLE-2670__braintree_virtual_terminal_ui_fix__4.1.0.patch) failed.
Error Output:
================
error: vendor/paypal/module-braintree-core/view/adminhtml/web/css/source/_module.less: already exists in working directory
Patch BUNDLE-2670 can't be applied to clean Magento instance
```

**Review by BarnyShergold — APPROVED** [review]

**Comment by BaDos** [comment]
@magterskine Please check your patch.
There is some conflicts.

```Applying patch BUNDLE-2670 (/app/vendor/magento/magento-cloud-patches/patches/BUNDLE-2670__braintree_virtual_terminal_ui_fix__4.1.0.patch) failed.
Error Output:
================
error: vendor/paypal/module-braintree-core/view/adminhtml/web/css/source/_module.less: already exists in working directory
Patch BUNDLE-2670 can't be applied to clean Magento instance
[2020-08-10 18:38:47] ERROR: [12] The command "php ./vendor/bin/ece-patches apply --no-interaction" failed. Patch MCLOUD-6659 has been applied
Title: Redis improvements
File: MCLOUD-6659__fix_L2_redis_cache__2.4.0.patch
Affected components: magento/framework
Error: patch conflict happened
Start of rollback
Patch MCLOUD-6659 has been reverted
End of rollback
Applying patch BUNDLE-2670 (/app/vendor/magento/magento-cloud-patches/patches/BUNDLE-2670__braintree_virtual_terminal_ui_fix__4.1.0.patch) failed.
Error Output:
================
error: vendor/paypal/module-braintree-core/view/adminhtml/web/css/source/_module.less: already exists in working directory
Patch BUNDLE-2670 can't be applied to clean Magento instance
returned non-zero exit status 12  Time 01:49
```

**Comment by shiftedreality** [comment]
@rganin please re-open this PR once it's ready for review

**Pull request closed without merging**